jest.config.js

```diff
@@ -21,4 +22,14 @@ module.exports = {
     moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
     testRegex: "src(.*)\\.test\\.tsx?$",
     testEnvironment: "node",
+    transform: {
+        "^.+\\.(t|j)s$": [
+            "@swc/jest",
+            {
+                jsc: {
+                    target: "es2021",
+                },
+            },
```

Comment:
As of https://github.com/swc-project/jest/pull/75, these are no longer required.

```suggestion
```